resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -46,7 +46,7 @@
 	 * @returns {jQuery} Returns a jQuery promise
 	 * @see https://developers.facebook.com/docs/javascript/quickstart/v2.2
 	 */
-	$.loadFacebookAPI = function (callback) {
+	$.loadFacebookSDK = function (callback) {
```

Comment:
@jsutterfield Good question. This was done originally to only show FB controls if FB is reachable and not blocked as in countries like China. This is why we only display them on successful FB library load. But again, we'll change this soon as described in SOC-240, but not yet :)